Automattic/woocommerce-payments
# Fix husky pre-commit hook

Fixes an issue where Husky isn't running pre-commit hooks due to necessary changes when migrating from v4 to v5, which we did [a while ago](https://github.com/Automattic/woocommerce-payments/pull/1321) 😬 


### Testing instructions

**Make sure you run `npm ci` before testing.**

#### Test JS pre-commit hook

* Make a change to any `.js`, `.ts`, or `.tsx` file that will be fixed by prettier.
* Try to commit the change
* Notice husky (or rather `lint-staged`) will prevent you from making an empty commit (because prettier fixed the issue during pre-commit)

#### Test PHP pre-commit hook

* Make a change to any `.php` file that produces an error with `phpcs`
* Try to commit the change
* Notice husky (or rather `lint-staged`) will prevent the commit because of a linter failure

-------------------
